test/orm/declarative/test_dc_transforms.py

```diff
@@ -565,6 +565,18 @@ class Engineer(MappedAsDataclass, Person):
             ),
         )
 
+    @testing.only_if(lambda: compat.py310, "python 3.10 is required")
```

Comment:
I think we can remove this, since compare has always been an option of dataclass

Reply:
Changed, thanks.